KS-SK1129-WHGR-Update-htaccess_protect_directory_by_password-for-Webhosting-TRANSLATION

Kazarus4 opened the pull request:
All translations are done

need proof for all subs except FRFR and FRCA, will need duplication for EN subs.

Commit:
All headers done

Commit:
Objective done

Commit:
prerequisites done

Commit:
DE done

Commit:
ENGB done

Commit:
ES subs done

Commit:
PL done

Commit:
PT done

Commit:
All translations done

Commit:
PL correction

Commit:
Links corrections

OVH-Team-Guides commented on pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md:
```diff
@@ -1,156 +1,269 @@
 >
 
-**Última actualización: 20/09/2021**
+**Última actualización: 01/06/2023"**
```
```suggestion
**Última actualización: 01/06/2023**

```

OVH-Team-Guides commented on pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md:
```diff
@@ -1,156 +1,269 @@
-- Disponer de las claves necesarias para conectarse al [espacio de almacenamiento del alojamiento](../conexion-espacio-almacenamiento-ftp-alojamiento-web/).
+- Tener un [plan de hosting](https://www.ovhcloud.com/es-es/web-hosting/).
+- Estar conectado a su [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es).
+- Disponer de las claves de conexión a [el espacio FTP de su alojamiento](/pages/web/hosting/ftp_connection).
```
```suggestion
- Disponer de las claves de conexión al [espacio FTP de su alojamiento](/pages/web/hosting/ftp_connection).

```

OVH-Team-Guides commented on pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md:
```diff
@@ -1,156 +1,269 @@
+>
+> Tenga en cuenta, por ejemplo, que si utiliza un **C**onent **M**anagement **S**ystem (**CMS**), existen otras soluciones de seguridad.
+>
+Si utiliza un CMS WordPress, OVHcloud también pone a su disposición un tutorial sobre cómo [utilizar el archivo htaccess con WordPress](/pages/web/hosting/htaccess_how_to_protect_wordpress).
```
```suggestion
Si utiliza un CMS WordPress, OVHcloud también pone a su disposición un tutorial sobre [cómo utilizar el archivo htaccess con WordPress](/pages/web/hosting/htaccess_how_to_protect_wordpress).

```

OVH-Team-Guides commented on pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md:
```diff
@@ -1,156 +1,269 @@
 
-Conéctese al [espacio de almacenamiento](../conexion-espacio-almacenamiento-ftp-alojamiento-web/) de su alojamiento. Abra el ["Pasta raiz"](../configurar-un-multisitio-en-un-alojamiento-web/#21-anadir-un-dominio-registrado-con-ovhcloud) de su sitio web.<br>
-Cree un archivo "crypter.php".
+- Crear archivos "crypt.php", ".htaccess" y ".htpasswd";
```
```suggestion
- Crear archivos "crypt.php", ".htaccess" y ".htpasswd".

```

OVH-Team-Guides commented on pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md:
```diff
@@ -1,156 +1,269 @@
+> Para modificar la versión de PHP de sus sitios web en su alojamiento web, consulte las siguientes guías:
+> 
+> - [Modificar la configuración de un alojamiento web](/pages/web/hosting/ovhconfig_modify_system_runtime)
+> - [Cambiar la versión de PHP de un alojamiento web](/pages/web/hosting/ovhconfig_modify_system_runtime/)
```
```suggestion
> - [Cambiar la versión de PHP de un alojamiento web](/pages/web/hosting/ovhconfig_modify_system_runtime)

```

OVH-Team-Guides replied:
Please fix this for all versions

OVH-Team-Guides commented on pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md:
```diff
@@ -1,156 +1,269 @@
+- **un único** fichero ".htaccess" y **un único** fichero ".htpasswd" por directorio o subdirectorio, para evitar conflictos entre diferentes ficheros ".htaccess" y diferentes ".htpasswd";
+- los archivos ".htaccess" y ".htpasswd" son invisibles para los internautas que visitan su sitio web;
+- las reglas declaradas en un archivo ".htaccess" se aplican a todo el directorio en el que está instalado el archivo ".htaccess", así como a todos los subdirectorios del mismo directorio.
+Los archivos ".htpasswd" y ".htaccess" pueden estar en carpetas diferentes. Solo puede utilizar un fichero ".htpasswd" para varios ".htaccess".
```
```suggestion
- Los archivos ".htpasswd" y ".htaccess" pueden estar en carpetas diferentes. Solo puede utilizar un fichero ".htpasswd" para varios ".htaccess".

```

OVH-Team-Guides commented on pages/web/hosting/htaccess_protect_directory_by_password/guide.es-us.md:
Please duplicate suggestions from the es-es file onto this es-us file.

Kazarus4 replied:
changed

OVH-Team-Guides commented on pages/web/hosting/htaccess_protect_directory_by_password/guide.fr-ca.md:
```diff
@@ -64,8 +64,8 @@ Nous allons vous expliquer les 4 étapes principales à réaliser pour protéger
-> - [Modifier la configuration de son hébergement web](/pages/web/hosting/ovhconfig_modify_system_runtime/)
-> - [Changer la version de PHP de son hébergement web](/pages/web/hosting/php_configure_php_on_your_web_hosting_2014/)
+> - [Modifier la configuration de son hébergement web](/pages/web/hosting/ovhconfig_modify_system_runtime)
+> - [Changer la version de PHP de son hébergement web](/pages/web/hosting/ovhconfig_modify_system_runtime/)
```
are you sure about this link ? (same as the one right above)

Kazarus4 replied:
/pages/web/hosting/php_configure_php_on_your_web_hosting_2014

OVH-Team-Guides commented on pages/web/hosting/htaccess_protect_directory_by_password/guide.fr-fr.md:
```diff
@@ -64,8 +64,8 @@ Nous allons vous expliquer les 4 étapes principales à réaliser pour protéger
-> - [Modifier la configuration de son hébergement web](/pages/web/hosting/ovhconfig_modify_system_runtime/)
-> - [Changer la version de PHP de son hébergement web](/pages/web/hosting/php_configure_php_on_your_web_hosting_2014/)
+> - [Modifier la configuration de son hébergement web](/pages/web/hosting/ovhconfig_modify_system_runtime)
+> - [Changer la version de PHP de son hébergement web](/pages/web/hosting/ovhconfig_modify_system_runtime/)
```
See comment on fr-ca above

Kazarus4 replied:
/pages/web/hosting/php_configure_php_on_your_web_hosting_2014

OVH-Team-Guides commented on pages/web/hosting/htaccess_protect_directory_by_password/guide.pt-pt.md:
Title and excerpt are not updated in PT and updated in other languages. Please check.

Kazarus4 replied:
done

Kazarus4 committed:
Update pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md

Co-authored-by: OVH-Team-Guides <43037584+OVH-Team-Guides@users.noreply.github.com>

Kazarus4 committed:
Update pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md

Co-authored-by: OVH-Team-Guides <43037584+OVH-Team-Guides@users.noreply.github.com>

Kazarus4 committed:
Update pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md

Co-authored-by: OVH-Team-Guides <43037584+OVH-Team-Guides@users.noreply.github.com>

Kazarus4 committed:
Update pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md

Co-authored-by: OVH-Team-Guides <43037584+OVH-Team-Guides@users.noreply.github.com>

Kazarus4 committed:
Update pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md

Co-authored-by: OVH-Team-Guides <43037584+OVH-Team-Guides@users.noreply.github.com>

Kazarus4 committed:
Update pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md

Co-authored-by: OVH-Team-Guides <43037584+OVH-Team-Guides@users.noreply.github.com>

Kazarus4 committed:
Update guide.fr-fr.md

/pages/web/hosting/php_configure_php_on_your_web_hosting_2014

Kazarus4 committed:
Update guide.fr-ca.md

/pages/web/hosting/php_configure_php_on_your_web_hosting_2014

Kazarus4 committed:
Update guide.pt-pt.md

Tutorial - Proteger um diretório ou a interface de administração do seu website por ficheiros .htaccess e .htpasswd

Descruba como proteger um diretório ou o acesso à administração do seu website através de uma autenticação com os ficheiros .htaccess e .htpasswd

Commit:
Update after YC proof done

Kazarus4 commented:
It stay to check ENGB and DE, duplication for EN subs and after all merge.

bs-ovh committed:
EN proof

bs-ovh committed:
DE proof

Commit:
EN duplications done
